hs-bindgen/src-internal/HsBindgen/Frontend/AST/Internal.hs

```diff
@@ -246,6 +247,9 @@ deriving instance Show (Id p) => Show (QualId p)
 deriving instance Eq   (Id p) => Eq   (QualId p)
 deriving instance Ord  (Id p) => Ord  (QualId p)
 
+instance (PrettyTrace (Id p)) => PrettyTrace (QualId p) where
```

Comment:
I wonder if we should rename `PrettyTrace`, since `QualId` is not a trace message :thinking: Not necessary to do as part of this PR though.

Reply:
Yes, I was also thinking the same. On the other hand, the `PrettyTrace` could also indicate how we should print it "in a trace message", and then it would be fine.